weblogic/hatch.toml

```
@@ -0,0 +1,8 @@
+[env.collectors.datadog-checks]
+
+[[envs.default.matrix]]
+python = ["27", "38"]
+version = ["12.2.1.4", "14.1.1.0-11"]
+
+[envs.default.overrides]
+matrix.version.env-vars = "WEBLOGIC_VERSION"
```

for easier selection, wdyt?

```suggestion
version = ["12", "14"]

[envs.default.overrides]
matrix.version.env-vars = [
  { key = "WEBLOGIC_VERSION", value = "12.2.1.4", if = ["12"] },
  { key = "WEBLOGIC_VERSION", value = "14.1.1.0-11", if = ["14"] },
]
```

---

Good idea.

Also applying this to https://github.com/DataDog/integrations-core/pull/12989/files